src/soter/openssl/soter_hash.c

```diff
@@ -61,7 +61,11 @@ soter_status_t soter_hash_update(soter_hash_ctx_t* hash_ctx, const void* data, s
         return SOTER_INVALID_PARAMETER;
     }
 
+#ifndef THEMIS_EXPERIMENTAL_OPENSSL_3_SUPPORT
     if (!EVP_MD_CTX_md(hash_ctx->evp_md_ctx)) {
+#else
+    if (!EVP_MD_CTX_get0_md(hash_ctx->evp_md_ctx)) {
```

Comment:
openssl 1.x doesn't have `EVP_MD_CTX_md`?

Reply:
It does. But it is also deprecated in OpenSSL 3